fabfile/setup.py

```diff
@@ -70,7 +70,7 @@ def common_setup(build_citus_func):
 
     prefix.check_for_pg_latest()
     # empty it but don't delete the link
-    run('rm -r {}/ || true'.format(config.PG_LATEST))
+    run('rm -r {}/* || true'.format(config.PG_LATEST))
```

Comment:
Is this to keep the folder there?

Reply:
As I know, If we want to `# empty it but don't delete the link`, we should call `rm -r softLink/*`. Otherwise, I get error for this `rm` command

Reply:
What error do you get? We use this function in other tests too, I can check if the error you get exists in other functions as well.

Reply:
If we call `rm -r` for a link, we get the following error `rm: cannot remove '<softLinkName>/': Not a directory`